src/components/FooterContainer/FooterContainer.tsx

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2019 - 2022 Gemeente Amsterdam
+import type { FC } from 'react'
+import {
+  Link,
+  Footer as StyledFooter,
+  FooterTop,
+  FooterBottom,
+  themeSpacing,
+  themeColor,
+} from '@amsterdam/asc-ui'
+import styled from 'styled-components'
+
+import { getIsAuthenticated } from 'shared/services/auth/auth'
+import configuration from 'shared/services/configuration/configuration'
+import useIsIncidentMap from 'hooks/useIsIncidentMap'
+import { useSelector } from 'react-redux'
+import { makeSelectIncidentContainer } from 'signals/incident/containers/IncidentContainer/selectors'
+
+const FooterWrapper = styled.div`
+  background-color: ${themeColor('tint', 'level1')};
+  margin: 0 auto;
+  max-width: 1400px;
+  width: 100%;
+  padding-top: 0;
+`
+
+const StyledFooterTop = styled(FooterTop)`
+  margin-top: 80px;
+  min-height: 80px;
```

Comment:
can you use themeSpacing to define `margin-top` and `min-height`?
Like this: `margin-top: ${themeSpacing(20)};` (units of 4 px)

Reply:
Since it was already there I didn't want to change everything. But will do!